config/locales/en.yml

```diff
@@ -2784,6 +2784,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   confirm_hub_change: "Are you sure? This will change your selected hub and remove any items in your shopping cart."
   confirm_oc_change: "Are you sure? This will change your selected order cycle and remove any items in your shopping cart."
   location_placeholder: "Type in a location..."
+  gmap_load_failure: "Unable to load map. Please check your network and allow 3rd party cookies."
```

Comment:
I think the error message is a bit confusing, maybe something like this would be better :
"Unable to load map. Please check your browser settings and allow 3rd party cookies for this website"

Reply:
Works for me. I made the change.